src/bootstrap/setup.rs

```diff
@@ -152,6 +152,9 @@ pub fn interactive_path() -> io::Result<Profile> {
         io::stdout().flush()?;
         let mut input = String::new();
         io::stdin().read_line(&mut input)?;
+        if input == "" {
+            panic!("EOF on stdin when asking question")
+        }
```

Comment:
This will panic if you press `Ctrl+D` on stdin. Can you exit with an error instead?
```
Please choose one (a/b/c/d): thread 'main' panicked at 'EOF on stdin when asking question', src/bootstrap/setup.rs:156:13
```
I'd rather it was something like
```
Please choose one (a/b/c/d):
error: got end-of-file instead of a profile, assuming you want to exit
```

Reply:
OK